# This message holds the readings from controller signal for claraquino.

uint32		timestamp_ms   # Timestap of the microcontroller, in millisecs since boot
int32		Steer_controller_signal  # PWM
float		Throttle_controller_signal # DAC
